src/hackstead/mod.rs

```diff
@@ -0,0 +1,27 @@
+use actix_web::{get, web, HttpResponse};
+use hcor::{ServiceError, Hackstead, UserContact};
+
+#[get("/hackstead/")]
+pub async fn get_hackstead(form: web::Json<UserContact>) -> Result<HttpResponse, ServiceError> {
+    log::debug!("servicing get_hackstead request");
+
+    let slack_id = form.slack().ok_or(ServiceError::bad_request("no slack"))?;
```

Comment:
this should work even without slack users.

Reply:
yes but prototyping